baseapi.py

```diff
@@ -115,4 +130,4 @@ def request(self, url, method, params=None, data=None):
         except ValueError:
             raise JSONDecodeError()   
 
-        return json['d']
+        return json
```

Comment:
['d'] represents for response data. In some cases, it's not json obj, just "OK" so I remove from here and handle the data in each method